grails-app/controllers/au/org/ala/volunteer/TutorialsController.groovy

```diff
@@ -218,6 +218,9 @@ class TutorialsController {
             if (session.hasProperty(SESSION_KEY_TUTORIAL_FILTER)) {
                 filterParams.putAll(session[SESSION_KEY_TUTORIAL_FILTER] as Map)
             }
+            if (params.boolean("admin") == true) {
```

Comment:
This might be by lack of familiarity with the system/intent here but admin status doesn't seem like something that should be trusted as an HTTP parameter.  Is there any way a non-admin could get the admin version of the filter by just appending this param to the URL?

Reply:
This is not admin status/auth. This is to view the administrative tutorials. Possibly bad choice in keyword.